service-food/src/main/kotlin/de/hpi/cloud/food/crawler/OpenMensaCrawler.kt

```diff
@@ -0,0 +1,71 @@
+package de.hpi.cloud.food.crawler
+
+import com.beust.klaxon.Klaxon
+import de.hpi.cloud.food.crawler.openmensa.CanteenData
+import de.hpi.cloud.food.crawler.openmensa.OpenMensaMeal
+import de.hpi.cloud.food.crawler.utils.openStreamWith
+import java.net.URI
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+class OpenMensaCrawler(
+    val canteenData: CanteenData
+) {
+    companion object {
+        private const val OPENMENSA_API_VERSION = "v2"
+
+        val BASE_URI: URI = URI("https://openmensa.org/api/$OPENMENSA_API_VERSION/")
+        val DATE_FORMAT_ISO8601 = DateTimeFormatter.ISO_LOCAL_DATE!!
+        val DATE_FORMAT_COMPACT = DateTimeFormatter.ofPattern("yyyyMMdd")!!
```

Comment:
I think it makes more sense to define this in `ParsedMensaMeal` where it's being used.